app/workers/education_form/templates/5495.erb

```diff
@@ -67,16 +65,6 @@ SSN:   <%= @applicant.veteranSocialSecurityNumber %>
 
 VA File Number: <%= @applicant.vaFileNumber %>
 
-Branch of Service:   <%= @applicant.serviceBranch %>
```

Comment:
@goldenmeanie what @lihanli said - you can see the contents of that field in https://github.com/department-of-veterans-affairs/vets-api/pull/971/files#diff-75ac655c065161bfe233a450450220edR78 for instance.